Docker/run.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/sh
 
+# push data to prometheus via pushgateway url
+function pushToPrometheus {
+  curl -s -m 10 --data-binary @- "$PUSHGATEWAY_URL/metrics/job/containerd$1" &>/dev/null &
+}
+
+# Get push gateway url and container image from env variable
+if [[ ! -z "$PUSHGATEWAY_URL" ]] && [[ ! -z "$CONTAINER_IMAGE" ]] && [[ !z "$SD_PIPELINE_ID" ]]; then
```

Comment:
I think `[[ ]]` is a bash (or modern shells) feature.

https://serverfault.com/questions/52034/what-is-the-difference-between-double-and-single-square-brackets-in-bash

Reply:
yes tested its working.

Reply:
I see, thanks. 
And I found typo: `[[ !z "$SD_PIPELINE_ID" ]]` -> `[[ ! -z "$SD_PIPELINE_ID" ]]` ?

Reply:
uwc, fixed typo